packages/create-hint/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     ],
     "references": [
         { "path": "../hint" },
-        { "path": "../utils" }
+        { "path": "../utils" },
+        { "path": "../utils-tests-helpers" }
```

Comment:
This probably came from re-running one of our scripts, but since it's cross-package let's either drop it from this PR or include it as a separate commit for merging.

Reply:
ok